ui/src/app/job-list/table/table.component.spec.ts

```diff
@@ -279,6 +293,66 @@ describe('JobsTableComponent', () => {
     });
   }))
 
+  it('should select multiple jobs on shift-click', async( () => {
+    fixture.detectChanges();
+    const jobCheckboxes = getJobCheckboxes();
+    jobCheckboxes[1].nativeElement.click();
+    testComponent.jobsTableComponent.updateCheckBoxSelection(jobs[3], shiftClick);
+    fixture.detectChanges();
+
+    expect(jobCheckboxes.map(c => c.nativeElement.checked))
+      .toEqual([false, true, true, true, false]);
+  }))
+
+  it('should select multiple jobs on shift-click (reverse order)', async( () => {
```

Comment:
nit: One last thing I just noticed - please drop the space after async( `async( ()` -> `async(()`

Reply:
well spotted – done